src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form"
+      data-qa="big" 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get"
+    >
+      <label>
+        <img 
+          class="icon" 
+          src="images/Search.svg" 
+          alt="search icon"
+        >
+        <input 
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
        >
        
        <input 
```